xml/Currency.xml

```diff
@@ -154,10 +154,13 @@
             </name>
         </codelist-item>
         <codelist-item>
-            <code>BYR</code>
+            <code>BYN</code>
             <name>
                 <narrative>Belarussian Ruble</narrative>
             </name>
+            <description>
+                <narrative>Previously BYR. Codelist updated 08-08-2018.</narrative>
```

Comment:
Codelist update timestamps are recorded via github timestamps… Also, [the codelist management page](http://reference.iatistandard.org/203/codelists/codelist-management/#updates-to-non-embedded-codelists) says that [the non-embedded codelist changelog](http://reference.iatistandard.org/203/upgrades/nonembedded-codelist-changelog/) is updated with this info.

I don’t think this information should be included in the codelist item description.

Reply:
Oh, in fact: In the case of a codelist code change, I think the correct approach would be to mark `BYR` as `status="withdrawn"` with a `withdrawal-date` of 2018-08-08 (or whenever it was actually withdrawn) and then add a new codelist item `BYN` and set the `activation-date` to the same date.

Then it could also be added to [the non-embedded codelist changelog](http://reference.iatistandard.org/203/upgrades/nonembedded-codelist-changelog/), but as mentioned above I’d personally be in favour of scrapping this.